data/default_hw_requirements.json

```diff
@@ -21,5 +21,11 @@
     "ram_mib": 16384,
     "disk_size_gb": 100,
     "installation_disk_speed_threshold_ms": 10
+  },
+  "edge-worker": {
+    "cpu_cores": 2,
+    "ram_mib": 8192,
+    "disk_size_gb": 15,
```

Comment:
is it 15 because of the rounding?

Reply:
yeah, in case it will be 15.96 don't want to block

Reply:
can you please add it in the commit message so we will remember why